features/addNewDockingStation.feature

```diff
@@ -0,0 +1,9 @@
+Feature: As a system administrator
+  I want to add new docking stations to the system
```

Comment:
Fajnie jak ten opis wyraża jaka jest potrzeba danej funkcji w systemie coś jak:

```feature
Feature: As a system administrator
        I want to add new docking stations
        so public transport users can search bikes in system including that station
```

chodzi o to że wtedy wiadomo jaki problem przez daną funkcje w systemie miał być rozwiązany